automation/bootstrap/install_openvino_package.sh

```diff
@@ -1,53 +1,14 @@
 #!/bin/bash
 
+set -e
 
-printf "\nDownload and install OpenVINO packages\n\n"
+OPENVINO_FOLDER="${HOME}/intel/openvino_2022/"
+OPENVINO_LIB_LINK="https://storage.openvinotoolkit.org/repositories/openvino/packages/2022.1/l_openvino_toolkit_runtime_ubuntu20_p_2022.1.0.643.tgz"
```

Comment:
Shouldn't we get package version from `openvino_version.yml`?

Reply:
is this approach supposed to work for Darwin / macOS use-cases?

Reply:
I don't think this will work for Darwin / macOS, but all bootstrap scripts need to be tested to work on those systems